includes/wikia/services/MediaQueryService.class.php

```diff
@@ -230,21 +232,21 @@ private function getMediaDataFromCache( Title $media, $length = 256 ) {
 				else {
 					$videoHandler = false;
 				}
-				$this->mediaCache[ $media->getDBKey() ] = array(
+				$this->mediaCache[ $cacheKey ] = array(
 					'title' => $media->getText(),
-					'desc' => $articleService->getTextSnippet( $length ),
+					'desc' => ( $length > 0 ? $articleService->getTextSnippet( $length ) : '' ),
 					'type' => ( $isVideo ? self::MEDIA_TYPE_VIDEO : self::MEDIA_TYPE_IMAGE ),
 					'meta' => ( $videoHandler ? array_merge( $videoHandler->getVideoMetadata(true), $videoHandler->getEmbedSrcData() ) : array() ),
 					'thumbUrl' => ( !empty($thumb) ? $thumb->getUrl() : false
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1125)